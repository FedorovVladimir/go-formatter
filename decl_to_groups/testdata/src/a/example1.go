package main

import ( // want "rm decl" "incorrect single declaration style"
	"fmt"
)

var ( // want "rm decl" "incorrect single declaration style"
	v1 = "v1"
)

const ( // want "rm decl" "incorrect single declaration style"
	c1 = "c1"
)

func main() {
	fmt.Println()
}
