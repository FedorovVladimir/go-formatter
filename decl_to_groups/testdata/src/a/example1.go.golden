package main

import (
	"fmt"
)

var (
	v1 = "v1"
)

const (
	c1 = "c1"
)

const (
	c3 = "c3"
	c4 = "c4"
)

const (
	c5 = "c5"

	c6 = "c6"
)

func main() {
	fmt.Println()
}
